src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.3.1.11724")]
+[assembly: AssemblyVersion("2.3.2.15684")]
```

Comment:
Revert these?

Reply:
This was actually meant to be submitted since the file was not updated locally correctly for me before. But I can revert for now and inspect build machine later

Reply:
Looks like the EngOps build failed.

Reply:
It is expected @reddyashish , see reason in original PR. The older build is depending on older version of .Net which is not on current build machine